build.sbt

```diff
@@ -943,6 +943,50 @@ def osgiTestProject(p: Project, framework: ModuleID) = p
     cleanFiles += (ThisBuild / buildDirectory).value / "osgi"
   )
 
+lazy val sbtTest = project.in(file("test") / "sbt-test")
+  .enablePlugins(ScriptedPlugin)
+  .settings(disableDocs)
+  .settings(
+    scalaVersion := "2.12.18",
```

Comment:
This should be the Scala version on which sbt is running. yoshord on discord suggested

> add sbt-buildinfo to the meta-metabuild, and generate a BuildInfo.scala for the metabuild to use